frontend/components/KolideAce/KolideAce.jsx

```diff
@@ -90,7 +90,10 @@ class KolideAce extends Component {
           maxLines={20}
           name={name}
           onChange={onChange}
-          onLoad={onLoad}
+          onLoad={(editor) => {
```

Comment:
lets pull this out into a named function. Things can get messy when we start passing anonymous functions into react components

Reply:
I fixed it--can you check my syntax/naming convention and modify it as needed?

Reply:
looks good 👍🏽